integration/kubernetes/helm-chart/alluxio/templates/worker/daemonset.yaml

```diff
@@ -88,6 +88,10 @@ spec:
       {{- if .Values.imagePullSecrets }}
 {{ include "alluxio.imagePullSecrets" . | indent 6 }}
       {{- end}}
+      initContainers:
```

Comment:
It is true that workers may have to restart once or twice waiting for master, but they will be fine eventually. Is this a concert to you?

Reply:
I know that they would be fine eventually. However, the restarts of pods may be treated as an indicator of the health of the whole system. Say, I may want to add metrics to record the restarts of pods or whatever.